pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "netconf_client"
-version = "3.1.3"
+version = "3.2.0.1"
```

Comment:
3-number versions are supported by more tools

```suggestion
version = "3.2.0"
```

Reply:
This was done by accident. The version number should of course remain three digits. Thank you for finding the typo. I fix it.